tests/src/test/scala/whisk/core/cli/test/WskWebActionsTests.scala

```diff
@@ -47,94 +47,16 @@ import whisk.core.entity.Subject
  * Tests web actions.
  */
 @RunWith(classOf[JUnitRunner])
-class WskWebActionsTestsV1 extends WskWebActionsTests {
```

Comment:
There are still two versions of web actions. Why remove the tests for both versions?